src/jsmediatags.js

```diff
@@ -220,6 +220,12 @@ class Config {
     });
   }
 
+  static setAdditionalXhrHeaders(additionalXhrHeaders: Array<[string, string]>) {
```

Comment:
Can you turn this into an object instead? `{ [key: string]: string }`.
Also, define it as a type in FlowTypes: `export type HttpHeaders = ...`